apps/test/unit/code-studio/components/lessonExtrasTestHelpers.js

```diff
@@ -70,39 +52,32 @@ export const bonusLevels = [
     levels: [
       {
         id: 23226,
-        levelId: 9,
-        name: 'courseC_artist_prog_challenge1',
-        type: 'Artist',
-        solutionImageUrl:
-          'https://d3p74s6bwmy6t9.cloudfront.net/80cc9bbdbd9a05c1a0cf03500b4eb38c=development/2091.png'
+        display_name: 'courseC_artist_prog_challenge1',
+        description: null,
+        thumbnail_url:
+          'https://d3p74s6bwmy6t9.cloudfront.net/80cc9bbdbd9a05c1a0cf03500b4eb38c=development/2091.png',
+        url:
+          'http://studio.code.org:3000/s/coursef-2019/stage/1/extras?level_name=courseC_PlayLab_events_challenge1',
+        perfect: false
       },
       {
         id: 23227,
-        levelId: 10,
-        name: 'courseB_maze_seq_challenge2',
-        type: 'Maze',
-        map: [
-          [0, 0, 0, 0, 0, 0, 0, 0],
-          [0, 1, 1, 1, 4, 1, 3, 0],
-          [0, 1, 1, 0, 1, 1, 4, 0],
-          [0, 1, 0, 1, 1, 0, 1, 0],
-          [0, 1, 1, 1, 0, 1, 1, 0],
-          [0, 1, 1, 0, 1, 1, 1, 0],
-          [0, 2, 1, 1, 1, 1, 1, 0],
-          [0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
are we losing test coverage for the lesson extras level to generate a thumbnail from the maze level contents?

Reply:
I removed this entirely (outside of tests too)  in favor of using thumbnail urls which the levelbuilder sets or the solution image for the level. Thoughts on that decision?

Reply:
my concern would be if there are some maze levels where we are relying on the existing implementation, that would lose thumbnails as a result of this change. is there any way to confirm whether this is the case?

Reply:
I'll look into it.